app/views/admin/badges/award.html.erb

```diff
@@ -0,0 +1,19 @@
+<h3>Award Badges</h3>
+<div class="alert alert-warning"><strong>Warning: Usernames are case-sensitive and must be all lowercase! This task will fail silently when supplied with incorrect
+    usernames.<strong></div>
+<%= form_with(url: admin_badge_achievements_award_badges_path, local: true) do |f| %>
+<div class="form-group">
+  <%= f.label :badge, "Badge:" %>
+  <%= f.select("badge", @badge.map { |badge| [badge.title, badge.slug] }, include_blank: true, class: "form-control") %>
+</div>
+<div class="form-group">
+  <%= f.label :usernames, "Usernames (Comma Delimited, All Lowercase, No @ Sign Required In Front Of Usernames)*" %>
```

Comment:
"@ Symbol" instead of "@ Sign"? Very minor, feel free to ignore 😄

Reply:
Changed @msarit! 😄